elmslie-android/src/main/java/vivid/money/elmslie/android/storestarter/ViewBasedStoreStarter.kt

```diff
@@ -0,0 +1,45 @@
+package vivid.money.elmslie.android.storestarter
+
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.coroutineScope
+import androidx.lifecycle.whenCreated
+import kotlinx.coroutines.launch
+import vivid.money.elmslie.android.processdeath.ProcessDeathDetector
+import vivid.money.elmslie.android.util.fastLazy
+import vivid.money.elmslie.core.config.ElmslieConfig
+import vivid.money.elmslie.core.store.Store
+
+@Deprecated("Use store start event instead of init event.")
```

Comment:
I suggest to improve the message. It seems misleading for me.
What about: "Prefer setting start event inside [storeFactory] of elmStore delegate"?

Reply:
will rework it